lib/openstudio/workflow/util/energyplus.rb

```diff
@@ -304,6 +275,7 @@ def self.add_energyplus_output_request(workspace, idf_object)
           allowed_objects << 'Output:Meter:Cumulative:MeterFileOnly'
           allowed_objects << 'Meter:Custom'
           allowed_objects << 'Meter:CustomDecrement'
+          allowed_objects << 'EnergyManagementSystem:OutputVariable'
```

Comment:
So I understand the rationale of allowing to define a new EMS:OutputVariable: after all, you aren't creating a new EMS program, just trying to report an existing one.

This EMS:OutputVariable then has to be also referenced in a regular Output:Variable for output to file. I suppose we could make sure this is the case, but since it's clearly an advanced usage to begin with, I reckon it's not really mandatory here.